docker_services_cli/cli.py

```diff
@@ -59,9 +47,55 @@ def add_env_output(ctx, *args, **kwargs):
                     env_set_command=env_set_command,
                 )
 
-        return update_wrapper(add_env_output, func)
+        return update_wrapper(_print_env_output, func)
+
+    return print_env_output
+
+
+def services_by_service_type(func):
```

Comment:
minor: below there is `collect_services_by_type` however this is called `services_by_services_type`. What about `services_by_type`?